autogen.sh

```diff
@@ -4,9 +4,13 @@ test -n "$srcdir" || srcdir=`dirname "$0"`
 test -n "$srcdir" || srcdir=.
 olddir=`pwd`
 
+cd $srcdir
+
 # Checkout and update submodules
-git submodule init
-git submodule update --recursive
+if test -d .git; then
```

Comment:
I don't think this repo has any submodules. You can probably remove these lines from any repos that don't need it